test/e2e/features/basic.feature

```diff
@@ -4,12 +4,18 @@ Feature: Basic test
     User explores some of the top-level CRC commands while going
     through the lifecycle of CRC.
 
-    @darwin @linux @windows
+    @darwin @linux
     Scenario: CRC version
         When executing "CRC_DISABLE_UPDATE_CHECK=true crc version" succeeds
```

Comment:
@adrianriobo Shouldn't we just have a version function as part of `crcsuite.go` so we don't have to maintain the env setting in the feature file?

Reply:
Yeah definitely, I would prefer move towards that direction instead of executing...

Reply:
I will change this